docs/src/architectures/pet.rst

```diff
@@ -5,12 +5,160 @@ PET
+
+
+However, with deep neural networks, the situation is drastically different. 
+The exact minimum of the loss is typically never achieved; instead, the model 
+asymptotically converges to it during fitting. It is essential to ensure that 
+the total number of epochs is sufficient for the model to approach the optimum 
+closely, thus achieving good accuracy.
+
+**In the case of PET, there is only one hyperparameter that MUST be manually 
+adjusted for each new dataset: the cutoff radius.** The selected cutoff 
+significantly impacts the model's accuracy and fitting/inference times, making 
+it very sensitive to this hyperparameter. All other hyperparameters can be safely set 
+to their default values. The next reasonable step (after fitting with default settings), especially for large datasets 
+is to try to increase the duration of fitting and see if it improves the accuracy of the obtained model. 
+
+Selection of ``R_CUT``
+*************************
+
+A good starting point is to select a cutoff radius that ensures about 20-30
+neighbors on average. This can be done by analyzing the neighbor lists for 
+different cutoffs before launching the training script. `This 
+<https://wiki.fysik.dtu.dk/ase/ase/neighborlist.html>`_ is an example of a 
+neighbor list constructor in Python.
+
+For finite configurations, such as small molecules in COLL/QM9/rmd17 datasets, 
+it makes sense to select ``R_CUT`` large enough to encompass the whole molecule. 
+For instance, it can be set to 100 Å, as there are no numerical instabilities 
+for arbitrarily large cutoffs.
+
+The hyperparameter for the cutoff radius is called ``R_CUT.``
+
+Selection of fitting duration
+*****************************
+
+The second most important group of settings is the one that adjusts the fitting duration 
+of the model. Unlike specifying a dataset-specific cutoff radius, this step is 
+optional since reasonable results can be obtained with the default fitting 
+duration. The time required to fit the model is a complex function of the 
+model's size, the dataset's size, and the complexity of the studied 
+interatomic interactions. The default value might be insufficient for 
+large datasets. If the model is still underfit after the predefined number of 
+epochs, the fitting procedure can be continued by relaunching the fitting 
+script with the same calculation name.
+
+However, the total number of epochs is only part of the equation. Another key
+aspect is the rate at which the learning rate decays. We use `StepLR
+<https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html>`_
+as a learning rate scheduler. This scheduler reduces the learning rate by a
+factor of `gamma` (`new_learning_rate = old_learning_rate * gamma`) every
+`step_size` epochs. In the current implementation of PET, `gamma` is fixed at 0.5,
+meaning that the learning rate is halved every `step_size` epochs.
+
+If `step_size` is set too small, the learning rate will decrease to very low
```

Comment:
```suggestion
If ``step_size`` is set too small, the learning rate will decrease to very low
```

Reply:
and more of the same above & below. (`gamma` as well)

Reply:
should be resolved now